tests/datasources/test_chirps.py

```diff
@@ -0,0 +1,360 @@
+"""Tests for the Chirps module."""
```

Comment:
The coverage overall is very good. The only thing really missing is testing a few edge cases where you raise an error. But you can skip it if you want.